src/V3Sched.cpp

```diff
@@ -490,9 +495,15 @@ AstNode* createInputCombLoop(AstNetlist* netlistp, AstCFunc* const initFuncp,
                                              ? extraTriggers.allocate("DPI export trigger")
                                              : std::numeric_limits<unsigned>::max();
     const size_t firstVifTriggerIndex = extraTriggers.size();
-    for (const auto& p : virtIfaceTriggers) {
+    for (const auto& p : virtIfaceTriggers.m_ifaceTriggers) {
         extraTriggers.allocate("virtual interface: " + p.first->name());
     }
+    const size_t firstVifMemberTriggerIndex = extraTriggers.size();
+    for (const auto& p : virtIfaceTriggers.m_memberTriggers) {
+        const auto& item = p.first;
+        extraTriggers.allocate("virtual interface member: " + item.m_ifacep->name() + "."
+                               + item.m_memberp->name());
+    }
```

Comment:
Without this (and it's copies), "member triggers" resolve to incorrect trigger indices.